pagopa/io-app
# chore(IT Wallet): [SIW-1367] Add credential details screen navigation from wallet

> [!WARNING]
> This PR depends on #5994

## Short description
This PR adds the ability to view the credential details from the wallet

## List of changes proposed in this pull request
- Added `WalletCardPressableBase` (previously `PaymentCardPressableBase`)
- Added `ItwPresentationCredentialDetailScreen` (previously `ItwPresentationEidDetailScreen`)
- Added `itwCredentialByTypeSelector` selector
- Removed unnecessary and/or broken playgrounds

## How to test
Tap on a credential in the wallet. You should be able to navigate to the details screen

## Preview

https://github.com/user-attachments/assets/2c2bcb3b-fc2c-41a0-9430-50dcc05690c3
